server/src/logic/handler/routes/recipe.handler.ts

```diff
@@ -31,4 +88,45 @@ export class RecipeHandler {
       })
       .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
   }
+
+  public update = (req: Request, res: Response): void => {
+    const newRecipe: Recipe = req.body as Recipe;
+
+    this._db.users.getUserById(newRecipe.userId)
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+
+    this._db.recipes.update(req.params.id, newRecipe)
+      .then(() => {
+        this.getById(req, res);
+      })
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+  }
+
+  public updateRecipe = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
This was also my first Idea but the function is only used in recipes and does not really some validation stuff. I kept it in the handler because I didn't want to create a separate middleware for one single function related to one special part of the app (recipe). What do you think, is it worth to create this new class for one function?